docs/user-guide.md

```diff
@@ -1,35 +1,52 @@
 # User Guide
 
-## Advanced
-
-### Building the base and node images
-#### Base image
-`kind` runs a local kubernetes cluster by using Docker containers as "nodes."
-`kind` uses the [`node-image`][node image] to run kubernetes artifacts, such 
-as `kubeadm` or `kubelet`.
-The `node-image` in turn is built off the [`base-image`][base image], which
-installs all the dependencies needed for Docker and Kubernetes to run in a
-container.
+## Creating a Cluster
 
-To build the `base-image` we use the `build` command:
+Creating a kubernetes cluster is as simple as:
 ```
-$ kind build base-image
+$ kind create cluster
 ```
 
-If you want to specify the path to the base image source files you can use the
-`--source` flag.
-If `--source` is not specified, `kind` has enbedded the contents of the in
-default base image in [`pkg/build/base/sources`][pkg/build/base/sources] and
-will use this to build it.
+This will bootstrap a kubernetes cluster using a pre-built 
+[node image][node image] - you can find it on docker hub
+[`kindest/node`][kindest/node]. 
+If you desire to build the node image yourself see the 
+[building image](#building-images) section.
+To specify another image use the `--image` flag.
 
-By default, the base image will be tagged as `kindest/base:latest`.
-If you want to change this, you can use the `--image` flag.
+By default, the cluster will be given the name `kind-1`. "1," here, is the
+default context name. 
+Use the `--name` flag to assign the cluster a different context name.
 
+Once a cluster is created you can use [kubectl][kubectl] to interact with it by
+using the configuration file generated by `kind`:
 ```
-$ kind build base-image --image base:v0.1.0
+export KUBECONFIG="$(kind get kubeconfig-path)"
+kubectl cluster-info
 ```
 
-#### Node image
+`kind get kubeconfig-path` returns the location of the generated confguration
+file.
+
+`Kind` also allows you to customize your kubernetes cluster by using a kind
```

Comment:
nit: stylized as `kind` :^) acronyms should be same case imo, and I think KIND looks a little shouty so I've avoided it for now :-)

Reply:
fixed haha